src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -146,14 +146,14 @@ public function dump(array $options = array())
         if (!empty($options['file']) && is_dir($dir = dirname($options['file']))) {
             // Build a regexp where the first root dirs are mandatory,
             // but every other sub-dir is optional up to the full path in $dir
-            // Mandate at least 2 root dirs and not more that 5 optional dirs.
+            // Mandate at least one parent dir and not more that 5 optional dirs.
 
             $dir = explode(DIRECTORY_SEPARATOR, realpath($dir));
             $i = count($dir);
 
             if (3 <= $i) {
```

Comment:
isn't this value supposed to be changed too?

Reply:
it isn't :)